doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -840,14 +808,8 @@ gls{PCM}
 
 \end_inset
 
- instance provided to Beagle will be found in the 
-\begin_inset CommandInset ref
-LatexCommand ref
-reference "result PCM"
-
-\end_inset
-
 .
+ It will only add information.
```

Comment:
1. That’s redundant, as it’s the double negation of the above. I think we can omit the sentence.

2. Please don’t switch the time, use “It only adds information”